jawampa-core/src/main/java/ws/wamp/jawampa/MessageInfo.java

```diff
@@ -0,0 +1,29 @@
+package ws.wamp.jawampa;
+
+public class MessageInfo<T> {
+	
+	
+	T message;
```

Comment:
Please make the class immutable. -> Make fields final and remove getters.
One instance will be shared by many subscribers, and immutable class means that one subscriber can't mess things up for the other subscribers.